src/java.base/share/classes/java/lang/constant/DynamicConstantDesc.java

```diff
@@ -56,7 +56,7 @@
  *
  * @since 12
  */
-public abstract class DynamicConstantDesc<T>
+non-sealed public abstract class DynamicConstantDesc<T>
```

Comment:
can you explain why `DynamicConstantDesc` is non-sealed?

Reply:
It's a permitted subclass of ConstantDesc, so it must be either `final`, `sealed`, or `non-sealed`. Making it `non-sealed` means it can still be extended.

Reply:
I should have made it clear.  I was expecting `DynamicConstantDesc` should be `sealed`.  Do you expect non-JDK implementation class extending `DynamicConstantDesc`?

Reply:
From the ConstantDesc Javadoc:
```
 * <p>Non-platform classes should not implement {@linkplain ConstantDesc} directly.
 * Instead, they should extend {@link DynamicConstantDesc} (as {@link EnumDesc}
 * and {@link VarHandleDesc} do.)
```

Reply:
Thanks.  I missed this javadoc.